requirements/edx/stanford.txt

```diff
@@ -16,3 +16,4 @@ ubcpi-xblock==0.5.3
 -e git+https://github.com/oppia/xblock.git@9f6b95b7eb7dbabb96b77198a3202604f96adf65#egg=oppia-xblock
 git+https://github.com/mfogel/django-settings-context-processor.git
 git+https://github.com/open-craft/problem-builder.git@v2.6.5#egg=xblock-problem-builder==2.6.5
+-e /edx/app/edxapp/xblocks/xblock-submit-and-compare
```

Comment:
@saharm1 This line should actually replace line 8 above. While we're installing our local copy, we do _not_ want to install the current/master version.

Reply:
Why is this change in this PR in the first place, @stvstnfrd ?

Reply:
@caesar2164 this was only so I could show @stvstnfrd my progress and check what wasnt working. Ill close it now.